components/feature/session/src/main/java/mozilla/components/feature/session/SessionUseCases.kt

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.session
+
+import mozilla.components.browser.session.SessionManager
+import mozilla.components.concept.engine.Engine
+
+/**
+ * Contains use cases related to the session feature.
+ */
+class SessionUseCases(
+    sessionManager: SessionManager,
+    engine: Engine,
+    sessionMapping: SessionMapping
+) {
```

Comment:
Yes, I made the wrapping class instantiable so we can initialize all use-cases (grouped together) in one go. It's based on the assumptions that all related use-cases have the same core dependencies. It seemed reasonable, but of course, this could turn out to be a wrong assumption. Then we can remove the wrapper.